Core/Core/Helpers/BinaryFormatter.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.IO;
+using System.Text;
+
+namespace Speckle.Core.Helpers;
+
+/// <summary>
+/// Mimics the behaviour of System.Runtime.Serialization.Formatters.Binary.BinaryFormatter for compatibility with Dotnet 8
+/// </summary>
+public static class BinaryFormatter
+{
+  /// <summary>
+  /// Leading bytes on a BinaryFormatter (should be consistent, though I haven't found an actual definition)
+  /// </summary>
+  private static readonly byte[] s_prefix = new byte[] { 0, 1, 0, 0, 0, 255, 255, 255, 255, 1, 0, 0, 0, 0, 0, 0, 0, 6, 1, 0, 0, 0 };
+  /// <summary>
+  /// Indicates end of file
+  /// </summary>
+  private const byte SUFFIX = 11;
+
+  /// <summary>
+  /// Mimics the behaviour of System.Runtime.Serialization.Formatters.Binary.BinaryFormatter().Serialize() when applied to a single string record.
+  /// </summary>
+  /// <param name="ms">Memory stream to append to</param>
+  /// <param name="value">String to serialise</param>
+  public static void SerialiseString(MemoryStream ms, string value)
```

Comment:
guessing there's not much we can do about these statics atm? extensions methods maybe an option?